data/channels.csv

```diff
@@ -10984,6 +10984,7 @@ KERADT1.us,KERA-DT1,,PBS,,US,US-TX,Dallas,s/US-TX,eng,education,FALSE,,,,http://
 KERADT2.us,KERA-DT2,,PBS Kids,,US,US-TX,Dallas,s/US-TX,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/7/76/PBS_Kids_Logo.svg/512px-PBS_Kids_Logo.svg.png
 KERADT3.us,KERA-DT3,,Create,,US,US-TX,Dallas,s/US-TX,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/7/71/Create_TV_network_logo.svg/512px-Create_TV_network_logo.svg.png
 KERADT4.us,KERA-DT4,,PBS World,,US,US-TX,Dallas,s/US-TX,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/6/6b/PBSworld.png
+Keralabhumi.in,Keralabhumi,,,,IN,,,c/IN,mal,general,FALSE,,,,https://keralabhumi.in/,https://i.imgur.com/CJpIMje.png

```

Comment:
Logo size should not exceed 512x512px.

```suggestion
Keralabhumi.in,Keralabhumi,,,,IN,,,c/IN,mal,general,FALSE,,,,https://keralabhumi.in/,https://i.imgur.com/05BslNS.png
```

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels